docs/examples/1_presentation/example_1_presentation_1_ping_user_service_udp.cpp

```diff
@@ -25,6 +25,8 @@
 #include <libcyphal/transport/udp/udp_transport.hpp>
 #include <libcyphal/transport/udp/udp_transport_impl.hpp>
 #include <libcyphal/types.hpp>
+
+#include <cassert>  // NOLINT for NUNAVUT_ASSERT
```

Comment:
I recall it was discussed somewhere that `serialization.hpp` needs to `#include <cassert>` so that the user code doesn't have to do it. We don't have an issue about this do we? If not, please make one.

Reply:
Created https://github.com/OpenCyphal/nunavut/issues/348